source/operations/monitoring/grafana.rst

```diff
@@ -0,0 +1,54 @@
+.. _minio-grafana:
+
+===================================
+Monitor a MinIO Server with Grafana 
+===================================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+   
+[Grafana](https://grafana.com/) allows you to query, visualize, alert on and understand your metrics no matter where they are stored. 
+Create, explore, and share dashboards with your team and foster a data driven culture.
+
+Prerequisites
+-------------
+
+- An existing :prometheus-docs:`Prometheus deployment <prometheus/latest/installation/>` with backing :prometheus-docs:`Alert Manager <alerting/latest/overview/>`
+- An existing MinIO deployment with network access to the Prometheus deployment
+- `Grafana installed <https://grafana.com/grafana/download>`__
+
+MinIO Grafana Dashboard
+-----------------------
+
+MinIO provides two official Grafana Dashboards you can download from the Grafana Dashboard portal.
+
+1. `MinIO Server metrics <https://raw.githubusercontent.com/minio/minio/master/docs/metrics/prometheus/grafana/minio-dashboard.json>`__
+2. `MinIO Bucket metrics <https://grafana.com/grafana/dashboards/19237-minio-bucket-dashboard//>`__
+
```

Comment:
Do we want to add a link to the JSON file with something like "You can track the changes to the Grafana dashboard by introspecting the JSON file in the MinIO Github repository:"

Reply:
So the purpose of these two links is to show our current version of the dashboards? Looking at the rendered page, I was not expecting them to be links to raw JSON files. 

I don't know much about Grafana, maybe that is a normal thing?

Reply:
Done something along these lines.

Reply:
I made these internal links to the two headings that follow, which makes much more sense in this context.